### PR TITLE
Add postgres and storage yaml

### PR DESCRIPTION
**Pull request by zhoujc999**

### CODE REVIEW

File: k8s/postgres_deployment_prod.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+              memory: "2Gi"
+          ports:
+            - containerPort: 5432
+          env:
+            - name: POSTGRES_USER
+              value: postgres
+            - name: POSTGRES_PASSWORD
+              value: postgres
+            - name: POSTGRES_DB
+              value: web_prod
```
Comment (zhoujc999) [inline k8s/postgres_deployment_prod.yaml:38]: @Jaxonwht 改一下？

Comment (zhoujc999) [inline k8s/postgres_deployment_prod.yaml:38]: @Jaxonwht 改一下？

Comment (Jaxonwht) [reply k8s/postgres_deployment_prod.yaml:38]: I thot we use expense db instead?

**Review by Jaxonwht — CHANGES_REQUESTED** [review]
Plan aborted

**Pull request closed without merging**